Hackney.Shared.Asset/Domain/AssetManagement.cs

```diff
@@ -13,10 +13,10 @@ public class AssetManagement
         public bool IsTMOManaged { get; set; }
         public string PropertyOccupiedStatus { get; set; }
         public string PropertyOccupiedStatusReason { get; set; }
-        public bool IsNoRepairsMaintenance { get; set; }
+        public bool? IsNoRepairsMaintenance { get; set; }
         public string CouncilTaxType { get; set; }
         public string CouncilTaxLiability { get; set; }
-        public bool IsTemporaryAccomodation { get; set; }
-        public bool ReadyToLetDate { get; set; }
+        public bool? IsTemporaryAccomodation { get; set; }
+        public bool? ReadyToLetDate { get; set; }
```

Comment:
Not sure why the type of this is bool by the way. Seems like it should be DateTime...